swift/ql/lib/codeql/swift/generated/ParentChild.qll

```diff
@@ -0,0 +1,2182 @@
+// generated by codegen/codegen.py
+import codeql.swift.elements
+
+private module Impl {
+  int getMaximumChildrenIndexElement(Element e) {
+    e = e and
+    result = 0
+  }
+
+  int getMaximumChildrenIndexCallable(Callable e) {
```

Comment:
Can all of these predicates be made private?

Reply:
sure, but they really cannot leak outside this file as they are in a private module, right?

Reply:
They can't leak outside the file, no. But if we mark them as `private` they also can't leak outside the module either. So that's even better 😄.